CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## v3.0.0 (unreleased)
+- Implement MultiUpsert method in client and expose it via the Client interface (#304)
```

Comment:
I don't see any breaking changes here, what's the reason for wanting to call this 3.0?

Reply:
i guess no reason. should i add a 2.7.0?